docs/index.rst

```diff
@@ -408,6 +408,29 @@ Use the following tokens for parsing and formatting. Note that they are **not**
 .. [#t4] timezone names from `tz database <https://www.iana.org/time-zones>`_ provided via dateutil package
 .. [#t5] this token cannot be used for parsing timestamps out of natural language strings due to compatibility reasons
 
+Built-in Formats
+++++++++++++++++
+
+There are several formatting standards that are provided as built-in tokens.
+
+.. code-block:: python
+
+    >>> arw=arrow.utcnow()
```

Comment:
Let's add a space to be consistent with the rest of the docs: `arw = arrow.utcnow()` :)